experiments/stv/gdaf-openstack/ansible/30-zeppelin-security.yml

```diff
@@ -40,3 +40,25 @@
         port_range_max: 8080
         remote_group: "{{ security['zeppelin'] }}"
 
+    # Allow any from masters.
+    - name: "Allow any from masters to Zeppelin"
+      os_security_group_rule:
+        cloud: "{{ cloudname }}"
+        state: present
+        security_group: "{{ security['zeppelin'] }}"
+        direction: 'ingress'
+        protocol:  'tcp'
+        remote_group: "{{ security['masters'] }}"
+
+    # Allow any from workers.
+    - name: "Allow any from workers to Zeppelin"
+      os_security_group_rule:
+        cloud: "{{ cloudname }}"
+        state: present
+        security_group: "{{ security['zeppelin'] }}"
+        direction: 'ingress'
+        protocol:  'tcp'
+        remote_group: "{{ security['workers'] }}"
```

Comment:
Allowing 'any' is very permissive. Is it possible to narrow this down to a port range ?

Reply:
Yes I'm sure we can narrow it down. Reason for the above (as well as the above ssh entry) was to try to get the version we have on the dev system in an automated scripts as it was, which we can then improve and change via configuration iteratively.

Reply:
OK, in which case we can approve this merge but add an issue to GitHub to fix this.